Bash-Release: 3.2
Patch-ID: bash32-002

Bug-Reported-by:    Jim Gifford <jim@jg555.com>
Bug-Reference-ID:   <12j2pc3aq35mb04@corp.supernews.com>
Bug-Reference-URL:
http://lists.gnu.org/archive/html/bug-bash/2006-10/msg00082.html

Bug-Description:

An incorrect encoding specification in the Content-Type header causes msgfmt
to fail, which causes make install' to fail.

--- bash-3.2/po/ru.po	2007-04-01 03:17:56.000000000 +0200
+++ bash-3.2/po/ru.po.new	2007-04-01 03:18:08.000000000 +0200
@@ -12,7 +12,7 @@
 "Last-Translator: Evgeniy Dushistov <dushistov@mail.ru>\n"
 "Language-Team: Russian <ru@li.org>\n"
 "MIME-Version: 1.0\n"
-"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Type: text/plain; charset=KOI8-R\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
 
